tests/nested/manual/tpm/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Check that tpm works properly on UC20
+
+description: |
+  This test check UC20 can boot with secure boot successfully and UC18 doesn't boot with secure boot
+
+systems: [ubuntu-18.04-*, ubuntu-20.04-*]
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    create_nested_core_vm true
+    start_nested_core_vm 
+
+restore: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    destroy_nested_vm
+    cleanup_nested_env
+
+execute: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    # Check tpm status on the nested vm
+    if is_core_20_nested_system; then
+        #TODO: check secure boot is enabled using test-snapd-mokutil snap
```

Comment:
Do you intend to address it in a followup? If so, then perhaps it would make sense to propose test-snapd-mokutil together with the followup - or at least build / install it as part of this PR to ensure it builds fine.

Reply:
I already pushed the snap, I'll add now the use of the snap, thanks for the suggestion

Reply:
Great, thanks